src/molgenis/capice/main_explain.py

```diff
@@ -0,0 +1,86 @@
+        self.log.info('Extracting total gain from model.')
+        feature_importances = model.get_booster().get_score(importance_type='total_gain')
+        self.log.debug('Extracted %d total_gain features from model.',
+                       len(feature_importances.keys()))
+        return feature_importances
+
+    def _extract_features_importances_weight(self, model: xgb.XGBClassifier):
+        self.log.info('Extracting weight from model.')
+        feature_importances = model.get_booster().get_score(importance_type='weight')
+        self.log.debug('Extracted %d weight features from model.',
+                       len(feature_importances.keys()))
+        return feature_importances
+
+    def _extract_features_importances_cover(self, model: xgb.XGBClassifier):
+        self.log.info('Extracting cover from model.')
+        feature_importances = model.get_booster().get_score(importance_type='cover')
+        self.log.debug('Extracted %d cover features from model.',
+                       len(feature_importances.keys()))
+        return feature_importances
+
+    def _extract_features_importances_total_cover(self, model: xgb.XGBClassifier):
+        self.log.info('Extracting total cover from model.')
+        feature_importances = model.get_booster().get_score(importance_type='total_cover')
+        self.log.debug('Extracted %d total_cover features from model.',
+                       len(feature_importances.keys()))
+        return feature_importances
+
+    def _convert_importances_to_dataframe(self, gain: dict, total_gain: dict, weight: dict,
+                                          cover: dict, total_cover: dict):
+        self.log.info('Converting importances to dataframe.')
+        feature_importances = pd.DataFrame(data=[gain.keys(), gain.values()],
+                                           index=['feature', 'gain']).T
+        feature_importances['total_gain'] = feature_importances['feature'].map(total_gain)
+        feature_importances['weight'] = feature_importances['feature'].map(weight)
+        feature_importances['cover'] = feature_importances['feature'].map(cover)
+        feature_importances['total_cover'] = feature_importances['feature'].map(total_cover)
+        self.log.debug('Converted %d features into the dataframe', feature_importances.shape[0])
+        self.log.debug('Converted all %d importance types into the dataframe',
+                       feature_importances.shape[1])
+        return feature_importances
+
+    def _order_importances(self, importances: pd.DataFrame):
+        self.log.info('Ordering feature importances.')
+        importances.sort_values(by='gain', ascending=False, inplace=True)
+
+    def _export(self, dataset, output):
+        output_path = os.path.join(output, CapiceManager().output_filename)
+        dataset.to_csv(output_path, compression='gzip', index=False, sep='\t')
+        self.log.info('Successfully exported explain to: %s', output_path)
```

Comment:
Process should print the path to the output file if the output path was not given by the user. Current behavior:
```
python capice.py explain -i v3.0.0-v1_grch37.pickle.dat

Process finished with exit code 0
```